.travis.yml

```diff
@@ -1,11 +1,39 @@
-language: minimal
+language: generic
 
-install:
-  - ./build.sh
+matrix:
+  include:
+    - os: linux
+      env:
+        - DOCKERIMAGE=condaforge/linux-anvil-aarch64
```

Comment:
This image is aarch64 and therefore you aren't hitting issues with constructor 3

Reply:
To run on x86 we added qemu to it